game/static/game/js/level_moderation.js

```diff
@@ -47,9 +47,9 @@ var saving = new ocargo.Saving();
 var CONFIRMATION_DATA = {
     'deleteLevel': {
         options: {
-            title: 'Delete level',
+            title: gettext('Delete level'),
         },
-        html: "<p>This student's level will be permanently deleted. Are you sure?</p>",
+        html: '<p>' + gettext("This student's level will be permanently deleted. Are you sure?") + '</p>',
```

Comment:
In general, there's no need to do this. Our translation software deals with non-text in translations strings fairly well